OpenFOAM/run.sh

```diff
@@ -1,96 +1,14 @@
 #!/bin/bash
 
-module load OpenFOAM/8-foss-2020a
-
-which ssh &> /dev/null
-if [ $? -ne 0 ]; then
-    # if ssh is not available, set plm_rsh_agent to empty value to avoid OpenMPI failing over it
-    # that's OK, because this is a single-node run
-    export OMPI_MCA_plm_rsh_agent=''
-fi
-
-source $FOAM_BASH
-
-if [ -z $EBROOTOPENFOAM ]; then
-    echo "ERROR: OpenFOAM module not loaded?" >&2
-    exit 1
-fi
-
-# Allow users to define the WORKDIR externally (for example a shared FS for multinode runs)
-export WORKDIR="${WORKDIR:-/tmp/$USER/$$}"
-echo "WORKDIR: $WORKDIR"
-mkdir -p $WORKDIR
-cd $WORKDIR
-pwd
-
-# motorBike, 2M cells
-BLOCKMESH_DIMENSIONS="100 40 40"
-# motorBike, 150M cells
-#BLOCKMESH_DIMENSIONS="200 80 80"
-
-# X*Y*Z should be equal to total number of available cores (across all nodes)
-X=${X:-4}
-Y=${Y:-2}
-Z=${Z:-1}
-# number of nodes
-NODES=${NODES:-1}
-# total number of cores
-NP=$((X * Y * Z))
-# cores per node
-PPN=$(((NP + NODES -1)/NODES))
-
-CASE_NAME=motorBike
-
-if [ -d $CASE_NAME ]; then
-    echo "$CASE_NAME already exists in $PWD!" >&2
-    exit 1
+# Allow oversubscription for OpenMPI since the workload is small and meant as proof of principle only 
+export OMPI_MCA_rmaps_base_oversubscribe=true
+
+if [[ $EESSI_CVMFS_REPO == "/cvmfs/software.eessi.io" ]] && [[ $EESSI_VERSION == "2023.06" ]]; then module load OpenFOAM/11-foss-2023a
```

Comment:
I would (only) load the OpenFOAM module in the script being called, not here.

Let's also add some indent to make this easier to parse, and maybe and extra `echo`:

```shell
if [[ $EESSI_CVMFS_REPO == "/cvmfs/software.eessi.io" ]] && [[ $EESSI_VERSION == "2023.06" ]]; then
    echo "Running demo for OpenFOAM v11..."
    ./bike_OpenFOAM_v11.sh

elif [ $EESSI_CVMFS_REPO == "/cvmfs/pilot.eessi-hpc.org" ]] && [[ $EESSI_PILOT_VERSION == "2021.12" ]]; then
    echo "Running demo for OpenFOAM v8..."
    ./bike_OpenFOAM_v8.sh

elif [ $EESSI_CVMFS_REPO == "/cvmfs/pilot.eessi-hpc.org" ]] && [[ $EESSI_PILOT_VERSION == "2023.06" ]]; then
    echo "There is no demo for OpenFOAM in /cvmfs/pilot.eessi-hpc.org/versions/2023.06" >&2
    echo "Please use the EESSI production repo /cvmfs/software.eessi.io" >&2
    exit 1
else
    echo "Don't know which OpenFOAM module to load for ${EESSI_CVMFS_REPO}/versions/${EESSI_VERSION}${EESSI_PILOT_VERSION}" >&2
    exit 1
fi